subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -734,11 +756,12 @@ class SubscriptionService(
                 scope_q, notif_attributes, notif_format, endpoint_uri, endpoint_accept, endpoint_receiver_info,
                 endpoint_notifier_info, status, times_sent, last_notification, last_failure, last_success, is_active, 
                 entity_selector.id as entity_id, id_pattern, entity_selector.type_selection as type_selection, georel,
-                geometry, coordinates, pgis_geometry, geoproperty, contexts
+                geometry, coordinates, pgis_geometry, geoproperty, contexts, throttling
             FROM subscription
             LEFT JOIN entity_selector ON entity_selector.subscription_id = subscription.id
             LEFT JOIN geometry_query ON geometry_query.subscription_id = subscription.id
             WHERE time_interval IS NOT NULL
+            AND throttling IS NULL
```

Comment:
if `timeInterval` is not null, `throttling` is null, so maybe this clause could be avoided?